app/assets/config/manifest.js

```diff
@@ -1,4 +1,4 @@
 //= link_tree ../images
-//= link_directory ../stylesheets .css
 //= link_tree ../../javascript .js
 //= link_tree ../../../vendor/javascript .js
```

Comment:
Shouldn't need JS stuff in sprockets config if it's being handled by importmap-rails. (And we may want to dump sprockets entirely and use propshaft.)